docs/reference/commandline/run.md

```diff
@@ -65,6 +65,7 @@ parent = "smn_cli"
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
       --privileged=false            Give extended privileges to this container
+      --read-bps-device=""       Specifies upper limit on READ rate from the device
```

Comment:
I think the words will be  too long if we want to describe  clearly arg fomat. How about just give an example like option --label